src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -431,13 +431,18 @@ public function makePathRelative($endPath, $startPath)
     /**
      * Mirrors a directory to another.
      *
+     * Copies files and directories from the origin directory into the target directory. By default:
+     *
+     *  - existing files in the target directory will be overwritten, except if they are newer (see the `override` option)
+     *  - files in the target directory that do not exist in the source directory will not be deleted (see the `delete` option)
+     *
      * @param string       $originDir The origin directory
      * @param string       $targetDir The target directory
-     * @param \Traversable $iterator  A Traversable instance
+     * @param \Traversable $iterator  By default all files and directories are copied, you can provide an iterator to filter which one to copy
```

Comment:
I would start by explaining what this parameter does, so something like: `Iterator that filters which files and directories to copy` which I think implies that the default is to copy everything.

Reply:
:+1: done